sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -56,9 +64,14 @@ CREATE OR REPLACE TABLE
           reported_metrics_ping BOOL
         OPTIONS
           (description = "True if the client ever reported a metrics ping."),
+          min_first_session_ping_submission_date DATE
```

Comment:
Could we have a schema.yaml file containing all the fields and their descriptions instead of having it all inside this init.sql?